python/python/glide/async_commands/transaction.py

```diff
@@ -1224,6 +1224,47 @@ def zrange_withscores(
 
         return self.append_command(RequestType.Zrange, args)
 
+    def zrank(
+        self: TTransaction,
+        key: str,
+        member: str,
+    ) -> TTransaction:
+        """
+        Returns the rank of `member` in the sorted set stored at `key`, with scores ordered from low to high.
+
+        See https://redis.io/commands/zrank for more details.
+
+        To get the rank of `member` with it's score, see zrank_withscore.
+
+        Args:
+            key (str): The key of the sorted set.
+            member (str): The member whose rank is to be retrieved.
+
+        Commands response:
+            Optional[int]: The rank of `member` in the sorted set.
+            If `key` does'nt exist, or if `member` is not present in the set, None will be returned.
+        """
+        return self.append_command(RequestType.Zrank, [key, member])
+
+    def zrank_withscore(
+        self: TTransaction,
+        key: str,
+        member: str,
+    ) -> TTransaction:
+        """
+        Returns the rank of `member` in the sorted set stored at `key` with it's score, where scores are ordered from the lowest to highest.
+        See https://redis.io/commands/zrank for more details.
```

Comment:
You usually have a blank line before `see`
```suggestion

        See https://redis.io/commands/zrank for more details.
```